filament/src/ColorGrading.cpp

```diff
@@ -76,30 +78,16 @@ ColorGrading::Builder& ColorGrading::Builder::whiteBalance(float temperature, fl
     return *this;
 }
 
-ColorGrading* ColorGrading::Builder::build(Engine& engine) {
-    return upcast(engine).createColorGrading(*this);
+ColorGrading::Builder& ColorGrading::Builder::channelMixer(
```

Comment:
This is basically a 3x3, so it's basically a colorspace transform :-)

Reply:
Pretty much :)